kanselarij-vlaanderen/frontend-kaleidos
# KAS-1637: Case Search Observer Refactor

# 👁️ Observer Refactors: Case Search

## What changed?

* Replaced the Observer with an action-based feedback from the searchTerm input field
* Refactored the entire component to be `@glimmer/object` instead.

## How to test?

We first thought the component was removed. It's not easy to find the location of this component at all! Go to Dossiers > Select a case > Select a subcase > Go to actions and click on move subcase. This opens a modal with a search. This is the case search component we've refactored in this PR! Everything should work as expected, of course 🌝
